packages/experiments/src/components/Shimmer/ShimmerRectangle/ShimmerRectangle.styles.ts

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import {
+  IStyle,
+  IStyleSet
+} from '../../../Styling';
+import { IStyleFunction } from '../../../Utilities';
+
+export interface IShimmerRectangle {
+
+}
+
+/**
+ * ShimmerRectangle component props.
+ */
+export interface IShimmerRectangleProps extends React.AllHTMLAttributes<HTMLElement> {
```

Comment:
Shouldn't this interface be in Shimmer.types.ts or in ShimmerRectangle.types.ts? What does it have to do with styles?

Reply:
Moved them in different file ShimmerRectangle.types.ts.

Reply:
Moved them in different files.